udsoncan/services/SecurityAccess.py

```diff
@@ -42,8 +42,8 @@ def make_request(cls, level, mode, key=None):
         :param mode: Type of request to perform. ``SecurityAccess.Mode.RequestSeed`` or ``SecurityAccess.Mode.SendKey`` 
         :type mode: SecurityAccess.Mode, int
 
-        :param key: When mode=``SendKey``, this value must be provided.
-        :type key: bytes
+        :param data

```

Comment:
Can you please add a description here to explains that this correspond to the `securityKey` or `securityAccessDataRecord`

Reply:
Sure.
I hope I won't make a mistake, it's my first pull request.